db/view/row_locking.cc

```diff
@@ -85,29 +85,21 @@ future<row_locker::lock_holder>
 row_locker::lock_ck(const dht::decorated_key& pk, const clustering_key_prefix& cpk, bool exclusive, db::timeout_clock::time_point timeout, stats& stats) {
     mylog.debug("taking shared lock on partition {}, and {} lock on row {} in it", pk, (exclusive ? "exclusive" : "shared"), cpk);
     auto tracker = latency_stats_tracker(exclusive ? stats.exclusive_row : stats.shared_row);
+    auto ck = cpk;
+    // The two-level lock is guaranteed to be kept alive as long as it's locked.
```

Comment:
nitpick: I would move this comment one line down, and start it with "the two_level_locks entry we just created  guaranteed to be kept alive ...".